pom.xml

```diff
@@ -85,6 +84,8 @@
         <brooklyn.version>1.0.0-SNAPSHOT</brooklyn.version><!-- BROOKLYN_VERSION -->
         <build.version>${revision}</build.version>
         <build.name>Apache Brooklyn</build.name>
+        <brooklyn.ui.relativePath.to.brooklyn.server.parent>../brooklyn-server/parent/</brooklyn.ui.relativePath.to.brooklyn.server.parent>
```

Comment:
The name is a bit too much for my taste @ahgittin, I would use a simpler `parent.relative.path`.
But happy to go ahead and merge if it works

Reply:
assuming this is set higher up in a complex project hierarchy it needs to be namespaced appropriately.  someone else might have a different `parent.relative.path`.  agree it's a little bit silly long but at least it's totally unambiguous. :)